README.md

```diff
@@ -23,25 +23,129 @@ import {
 } from 'react-native';
 import InfiniteScrollView from 'react-native-infinite-scroll-view';
 
+class ExampleComponent extends React.Component {
+  async _loadMoreContentAsync() {
```

Comment:
write this as 
```js
_loadMoreContentAsync = async () => {
};
```
and then it will bind at instantiation time, no need to call `bind(this)` on each render call anymore.

Might also want to make the comment more explanatory, say that it should eventually call `this.setState()` to update the component once the data is fetched, I could see people wondering if `_loadMoreContentAsync` should return the new data instead.